### ISSUE TITLE
[Windows] HAS_SELECTION error

### ISSUE THREAD

**Issue by kamyker** [tomrijndorp/vscode-finditfaster#75]
If I select text and click ctrl+shift+j or u it doesn't work.

```
PS H:\MyPath> HAS_SELECTION=1 c:\MyPath\.vscode\extensions\tomrijndorp.find-it-faster-0.0.30\find_within_files.ps1  'h:\MyPath'
HAS_SELECTION=1 : The term 'HAS_SELECTION=1' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the 
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ HAS_SELECTION=1 c:\MyPath\.vscode\extensions\tomrijndorp.find-it ...
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (HAS_SELECTION=1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Comment by Luiggi33** [comment]
This is because on windows, you can't set environment vars as on Linux. I addressed this in https://github.com/tomrijndorp/vscode-finditfaster/pull/74

**Comment by tomrijndorp** [comment]
Thank you @Luiggi33! This fix is now being published and is incorporated in version 0.0.31. @kamyker please let us know if there's still an issue :)
